### PR TITLE
Optimize the menu bar for macOS

### PR DESCRIPTION
**Pull request by X1nto**

### CODE REVIEW

File: src/main/mainWindow.ts [hunk 1]
```diff
@@ -182,13 +182,31 @@ function initMenuBar(win: BrowserWindow) {
             }
         },
         isDarwin && {
```
Comment (Vendicated) [inline src/main/mainWindow.ts:184]: its way less ugly if u do it like this
```js
...(!isDarwin ? [] : [
	all the stuff here
])
```

File: src/main/mainWindow.ts [hunk 2]
```diff
@@ -181,14 +183,34 @@ function initMenuBar(win: BrowserWindow) {
...
-        },
+        ...(!isDarwin ? [] : [
+            {
+                type: "separator"
+            },
+            {
+                label: "Settings",
+                accelerator: "CmdOrCtrl+,",
+                async click() {
+                    mainWin.webContents.executeJavaScript("Vencord.Webpack.Common.SettingsRouter.open()")
```
Comment (Vendicated) [inline src/main/mainWindow.ts:194]: did you even test this? this gives you a blank screen
```suggestion
                    mainWin.webContents.executeJavaScript("Vencord.Webpack.Common.SettingsRouter.open('My Account')")
```

Comment (X1nto) [reply src/main/mainWindow.ts:194]: Worked fine for me, weird

Comment (X1nto) [reply src/main/mainWindow.ts:194]: Just realized this seems to open whatever the last page in the settings was, so no wonder it displays a blank screen on the first try.

**Comment by Vendicated** [comment]
lint is failing

**Pull request merged**